### PR TITLE
slick scala3 support

### PR DESCRIPTION
**Pull request by jtjeferreira**

### CODE REVIEW

**Review by lloydmeta — APPROVED** [review]
LGTM. Just waiting for the weekend to merge + release.

File: build.sbt [hunk 1]
```diff
@@ -613,9 +613,10 @@ lazy val enumeratumSlick =
       version            := "1.7.5-SNAPSHOT",
       crossScalaVersions := scalaVersionsAll,
       libraryDependencies ++= Seq(
-        ("com.typesafe.slick" %% "slick" % "3.4.1").cross(CrossVersion.for3Use2_13),
+        ("com.typesafe.slick" %% "slick" % "3.5.0-M4"),
```
Comment (jtjeferreira) [inline build.sbt:616]: Please note this depends on a milestone version, so we can wait a bit

Comment (lloydmeta) [reply build.sbt:588]: I see we'er at M5 now; let's wait until we get to a stable release.

**Comment by jtjeferreira** [comment]
@lloydmeta can you have a look?

**Review by lloydmeta — APPROVED** [review]
LGTM, not merging yet as I don't have bandwidth this week to release.

**Comment by lloydmeta** [comment]
Thanks for this contribution and apologies for the long wait. Just released (pushed) 1.7.5 of `enumeratum-slick` to Maven central.

**Pull request merged**